Manifests/ManagedPreferencesApplications/com.google.Chrome.plist

```diff
@@ -6956,6 +7095,8 @@ URL patterns in this policy should not clash with ones configured via WebUsbAskF
 			<string>array</string>
 		</dict>
 		<dict>
+			<key>pfm_default</key>
+			<false/>
```

Comment:
Recent PR bumped the Chrome pfm_version, so this should be `6` to be inline with the current.  Since preferences weren't added here no need to bump further to `7`.

Reply:
That too is implemented in the PR on the same commit I mentioned above, b31199e.